bindings/cs/MongoDB.Libmongocrypt/Library.cs

```diff
@@ -562,8 +572,12 @@ internal class Delegates
             public delegate bool mongocrypt_ctx_decrypt_init(ContextSafeHandle handle, BinarySafeHandle binary);
 
             [return: MarshalAs(UnmanagedType.I1)]
-            public delegate bool
-                mongocrypt_ctx_explicit_encrypt_init(ContextSafeHandle handle, BinarySafeHandle binary);
+            public delegate bool mongocrypt_ctx_explicit_encrypt_init(ContextSafeHandle handle, BinarySafeHandle binary);
+            /// <summary>
```

Comment:
blank line before summary?

Does `mongocrypt_ctx_explicit_encrypt_init ` mentioned in summary refer to `mongocrypt_ctx_explicit_encrypt_expression_init` ?

Reply:
good point, fixed